### PR TITLE
Create Items.md

### PR DESCRIPTION
**Pull request by thejereman13**
Not quite perfect(I'm sure I missed something and will facepalm later when I find it). Let me know if I should cut some things out/add some. Figured I would at least detail the basics.

### CODE REVIEW

**Comment by cpw** [comment]
OK, this is a little bit too much of a "how to code java" tutorial. A lot of the content is good, but please, you don't have to explain how subclasses work..

**Comment by thejereman13** [comment]
Yeah, looking back, it seems I got a little carried away. I'll go remove a few lines and phrases.

**Comment by tterrag1098** [comment]
I feel like now there is not enough detail. Also you are lacking some headers which could better split up the article - remember, headers make links, so that people can easily jump between sections.

**Comment by LexManos** [comment]
itemVariable.getUnlocalizedName().substring(5)
NO

**Comment by tterrag1098** [comment]
Indeed, that is a Pahimar-ism that needs to die.

**Comment by tterrag1098** [comment]
Honestly it seems the author doesn't quite know the system well enough to document it, so I think this should be closed barring a total rewrite.

File: gettingstarted/Items.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+Making an Item
+--------------
+
+Any item, modded or vanilla, added to the minecraft world is an extension of the `Items` class from Minecraft. Start by declaring a variable of type Item, setting it as `new Item()`, thus telling minecraft that this will be an item. If you need help with (or don't know) proper naming schemes/conventions, see the Structuring documentation. The next step is to add/set an unlocalized name for your item. Unlocalized names are the strings Minecraft uses for lang files internaly. Unlocalized names are set on an item with the `.setUnlocalizedName(String)` function of the Item class; If you have a creative tab, you can put that declaration here as well (`.setCreativeTab()`).
+
+If your item is ever going to have functionality other than as a crafting component (see above note in Structuring), you will need to make your own class extending `Item` and then use that class in place of `Item()` when making the item. e.g. `testItem = new Item()` becomes `testItem = new testItemClass()`. More on extended Item classes below.
```
Comment (tterrag1098) [inline gettingstarted/Items.md]: This is still too java-tutorial-esque, you do not need to dumb it down this much, we can expect people to know what class extending is.

**Comment by tterrag1098** [comment]
All in all, I think Items deserve much more work than this, even if it may be a decent start. Pages on different methods, registration, different super classes for items, etc would all be good, instead of squishing it all into one incomplete page.

**Comment by LexManos** [comment]
Yes, Items should be a category. Start with 'The Basics' and move on.

**Comment by tterrag1098** [comment]
@thejereman13 Let us know your intentions, so we can either let you finish or let someone else take over the job

**Comment by thejereman13** [comment]
At this point, I have too much work on my plate to attend to this documentation. Many of your comments are beneficially constructive, however I concede this task, as my time is better spent elsewhere.
